eval/src/test_seq.rs

```diff
@@ -2,6 +2,7 @@ use std::marker::PhantomData;
 use time::{Duration};
 use rand::{StdRng};
 use actions::{Creator,Editor,Computor,Testor};
+use iodyn::finite_map::{Graph,FinMap};
```

Comment:
I doubt we need this dependency, since nothing else was added to the file. The test description here is generic over some high-level traits, such as the `EditInsert` that was defined above.